guides/tutorials/mediation.md

```diff
@@ -0,0 +1,120 @@
+
+Such statements adopt a useful level of abstraction--one that's highly recommended for most discussions. However, they make a number of simplifications. By modeling the roles of mediators and relays in routing, we can support routes that use multiple transports, routes that are not fully known (or knowable) to the sender, routes that pass through mix networks, and other advanced and powerful concepts."[Aries RFC 0046: Mediators and Relays](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0046-mediators-and-relays/README.md#aries-rfc-0046-mediators-and-relays)
+
+:::
+
+A practical example where the use of mediators is beneficial can be found in a mobile environment. A mobile agent, aka. your mobile wallet, receives messages from other agents. Technically, sending HTTP messages to a mobile agent is impossible because the agent does not have a public IP. As you have probably seen in previous examples, a workaround for this scenario is to use ngrok. Another option is to use a mediator. When an agent is configured to use a mediator, messages to that agent are routed through the mediator and from there to its destination. Thus, a mobile agent can use a "public" mediator to receive messages on its behalf and poll them from time to time. Here WebSockets come in handy because instead of polling messages from the mediator, the mediator can use WebSockets to push messages to the mobile agent. This is just one example of how a mediator can be used. Further resources about mediators can be found [here](./mediation.md#useful-resources).
+
+### 1. Agent Configuration
+
+First, we need to setup the agents.
+Depending on your target, [React
+Native](../getting-started/installation/react-native) or
+[Node.js](../getting-started/installation/react-native), this might vary.
+
+This tutorial assumes a [Node.js environment](../getting-started/installation/nodejs) for all agents (_Alice_, _Bob_, _Meditator_).
+
+To set up the agents, we use the `startAgent(...)` utility function. The function takes an object as argument containing the `name`, `port`, and `modules` of the agent to be created. In this example, the `modules` property is used to configure the `RecipientModule`, which is required to set up a mediator for an agent. Therefore, only Alice must provide this property.
+
+```typescript showLineNumbers mediation.ts section-1
+
+```
+
+:::tip
+
+In a production environment, it is recommendable to register a `WsInboundTransport()` for Alice, and a `WsOutboundTransport()` for the mediator. This allows the mediator to push messages to Alice, instead of having Alice to pull messages from the mediator all the time.
+:::
+
+### 2. Mediator Set Up
```

Comment:
In examples I would avoid variable name such as `medOob` as it can be a bit vague. I'm also a fan of using fully written out variable names (as people will often just copy what you write in the docs)

Reply:
I'll fix this